xBrokerConnector/Dockerfile

```
@@ -0,0 +1,17 @@
+# 1st Stage - Build
+FROM maven:3.6-openjdk-11 AS bluider
```

Probably that is what was wrong with it 🙂 
```suggestion
FROM maven:3.6-openjdk-11 AS builder
```